lib/src/serializers/json_api.dart

```diff
@@ -19,12 +19,15 @@ class JsonApiSerializer implements Serializer {
   @override
   JsonApiManyDocument deserializeMany(String payload) {
     Map<String, dynamic> parsed = parse(payload);
-    var docs = (parsed['data'] as Iterable).map((item) => JsonApiDocument(
-        item['id'],
-        item['type'],
-        item['attributes'],
-        item['relationships'],
-        parsed['included']));
+    var docs = (parsed['data'] as List)
```

Comment:
@halil81 Please reinstate `as Iterable` in this case. We're navigating a structure created by `dart:convert` so we can't make assumptions on the exact type.

Reply:
Thanks! @algodave, docs reinstated as `Iterable`